apps/api/src/app/environments/environments.controller.ts

```diff
@@ -116,6 +118,24 @@ export class EnvironmentsController {
     return await this.getApiKeysUsecase.execute(command);
   }
 
+  @Get('/api-keys/regenerate')
```

Comment:
Please change the endpoint method from GET to POST. GET methods should only retrieve data and not have another side effect and the implementation of this endpoint is modifying the resource in the database.